### PR TITLE
#161714413 Handle non-existent routes

### PR DESCRIPTION
**Pull request by victor-nweze-deel**
#### What does this PR do?
- Handle non-existent routes with a descriptive message

#### Description of Task to be completed?
- Create a route handler for all non-existent routes and HTTP methods

#### How should this be manually tested?
- Git clone this repository (Run `git clone https://github.com/andela/haven-ah-backend.git`)
- Navigate into the haven-ah-backend folder (Run `cd haven-ah-backend`)
- Start postman locally
- Start the development server (Run `npm run start-dev`)
- Fire up a non-existent route on postman

#### Any background context you want to provide?
- The application previously couldnt handle non-existent routes

#### What are the relevant pivotal tracker stories?
[#161714413](https://www.pivotaltracker.com/story/show/161714413)

#### Screenshots (if appropriate)
screen shot 2018-11-05 at 1 24 10 pm

### CODE REVIEW

**Review by TheoOkafor — COMMENTED** [review]
Nice work Victor

**Comment by PriscillaSam** [comment]
Nice job Victor.

File: server/routes/index.js [hunk 1]
```diff
@@ -2,9 +2,16 @@ import { Router } from 'express';
 
 import userRouter from './users';
 import articleRouter from './article';
+import { goodHttpResponse } from '../utilities/httpResponse';
+
 
 const router = Router();
 
+router.get('/', (request, response) => {
```
Comment (PriscillaSam) [inline server/routes/index.js]: @ProfJigsaw please why did you add this here as well?

Comment (victor-nweze-deel) [reply server/routes/index.js]: We need to handle the get request to the first version of the app.

Comment (seyi-adeleke) [reply server/routes/index.js]: Not sure I follow, is this with regards to api versioning?

**Comment by seyi-adeleke** [comment]
Yo conflicts @ProfJigsaw

File: server/routes/index.js (hunk 1 above)
Comment (victor-nweze-deel) [reply server/routes/index.js]: Yes it is

**Comment by victor-nweze-deel** [comment]
> Yo conflicts @ProfJigsaw

Will fix this now

File: server/routes/index.js (hunk 1 above)
Comment (seyi-adeleke) [reply server/routes/index.js]: we don't need to do this yet. please remove

**Comment by seyi-adeleke** [comment]
15 file changes? 😨

**Comment by victor-nweze-deel** [comment]
> 15 file changes? 😨

Sorry in trying to fix the merge conflict I accidentally squashed in an older commit message. I have fixed it

File: server/routes/index.js (hunk 1 above)
Comment (victor-nweze-deel) [reply server/routes/index.js]: Removed

**Review by seyi-adeleke — APPROVED** [review]
